src/main/java/com/archisacadeny/course/CourseRepository.java

```diff
@@ -249,4 +254,66 @@ public List<Course> getCoursesByInstructorId(long instructorId){
         return courses;
     }
 
+    public List<Course> getStudentEnrolledCourses(int studentId) {
+        // RETURNS COURSE ID s FOR NOW
+        String query = "SELECT course_id, name, instructor_id, credits, number,department,max_students FROM \"course_student_mapper\" " +
+                "LEFT JOIN \"courses\" ON course_student_mapper.course_id = \"courses\".\"id\""+
+                "WHERE student_id = '" + studentId + "'" ;
+
+        ArrayList<Course> courses = new ArrayList<>();
+
+        try (PreparedStatement statement = DataBaseConnectorConfig.getConnection().prepareStatement(query, ResultSet.TYPE_SCROLL_SENSITIVE,
+                ResultSet.CONCUR_UPDATABLE)) {
+            statement.execute();
+            ResultSet rs = statement.getResultSet();
+            Instructor instructor = new Instructor();
+            while (rs.next()) {
+                instructor.setId(rs.getLong("instructor_id"));
+                courses.add(new Course(rs.getInt("course_id")
```

Comment:
I think instead of constructor you can use getter and setter methods for data mapping